### ISSUE TITLE
incorrect AS3 declaration in f5-payg-autoscale-bigip-waf-nlb.template

### ISSUE THREAD

**Issue by chen23** [F5Networks/f5-aws-cloudformation#61]
## Description
when deploying f5-payg-autoscale-bigip-waf-nlb.template from: 

https://github.com/F5Networks/f5-aws-cloudformation/blob/ada5cd3a9d78ce40f404e020edf9245a22a30e30/experimental/autoscale/waf/via-nlb/1nic/existing-stack/payg/f5-payg-autoscale-bigip-waf-nlb.template

Commit: ada5cd3a9d78ce40f404e020edf9245a22a30e30

the custom-config.sh fails to complete due to an invalid AS3 declaration being used

here is the AS3 declaration that is being generated

```
{
  "class": "ADC",
  "schemaVersion": "3.0.0",
  "label": "autoscale_waf",
  "id": "AUTOSCALE_WAF",
  "remark": "Autoscale WAF",
  "waf": {
    "class": "Tenant",
    "Shared": {
      "class": "Application",
      "template": "shared",
      "serviceAddress": {
        "class": "Service_Address",
        "virtualAddress": "0.0.0.0"
      },
      "policyWAF": {
        "class": "WAF_Policy",
        "file": "/config/cloud/asm-policy-linux-high.xml"
      }
    },
    "http": {
      "class": "Application",
      "template": "http",
      "serviceMain": {
        "class": "Service_HTTP",
        "virtualAddresses": [
          {
            "use": "/waf/Shared/serviceAddress"
          }
        ],
        "serverTLS": {
          "bigip": "/Common/example-clientssl-profile"
        },
        "snat": "auto",
        "securityLogProfiles": [
          {
            "bigip": "/Common/Log illegal requests"
          }
        ],
        "pool": "pool",
        "policyWAF": {
          "use": "/waf/Shared/policyWAF"
        },
        "virtualPort": 443
      },
      "pool": {
        "class": "Pool",
        "monitors": [
          "http"
        ],
        "members": [
          {
            "autoPopulate": true,
            "hostname": "www.example.com",
            "servicePort": 80,
            "addressDiscovery": "fqdn"
          }
        ]
      }
    }
  }
}
```
here is what it should be (note https / HTTPS instead http/HTTP):
```
{
  "class": "ADC",
  "schemaVersion": "3.0.0",
  "label": "autoscale_waf",
  "id": "AUTOSCALE_WAF",
  "remark": "Autoscale WAF",
  "waf": {
    "class": "Tenant",
    "Shared": {
      "class": "Application",
      "template": "shared",
      "serviceAddress": {
        "class": "Service_Address",
        "virtualAddress": "0.0.0.0"
      },
      "policyWAF": {
        "class": "WAF_Policy",
        "file": "/config/cloud/asm-policy-linux-high.xml"
      }
    },
    "http": {
      "class": "Application",
      "template": "https",
      "serviceMain": {
        "class": "Service_HTTPS",
        "virtualAddresses": [
          {
            "use": "/waf/Shared/serviceAddress"
          }
        ],
        "serverTLS": {
          "bigip": "/Common/example-clientssl-profile"
        },
        "snat": "auto",
        "securityLogProfiles": [
          {
            "bigip": "/Common/Log illegal requests"
          }
        ],
        "pool": "pool",
        "policyWAF": {
          "use": "/waf/Shared/policyWAF"
        },
        "virtualPort": 443
      },
      "pool": {
        "class": "Pool",
        "monitors": [
          "http"
        ],
        "members": [
          {
            "autoPopulate": true,
            "hostname": "www.example.com",
            "servicePort": 80,
            "addressDiscovery": "fqdn"
          }
        ]
      }
    }
  }
}
```
## Template
For bugs, enter the template with which you are experiencing issues below.

 f5-payg-autoscale-bigip-waf-nlb.template from: 

https://raw.githubusercontent.com/F5Networks/f5-aws-cloudformation/master/experimental/autoscale/waf/via-nlb/1nic/existing-stack/payg/f5-payg-autoscale-bigip-waf-nlb.template

## Severity Level
For bugs, enter the bug severity level. Do not set any labels.

Severity: 2

**Comment by edwoodjrjr** [comment]
Tracking this with internal issue #849.

**Comment by edwoodjrjr** [comment]
This was fixed in release v4.1.1.
